### ISSUE TITLE
Sync `resource.stats` with the standards

### ISSUE THREAD

**Issue by krassowski** [frictionlessdata/frictionless-py#868]
# Overview

Consider introducing the `Stats` class (`resource.stats`)

---

```python
from frictionless import Resource

Resource(stats={'hash': '2a53375ff139d9837e93a38a279d63e5', 'bytes': 12345}).to_json('test.json')
```

```bash
$ cat test.json 
```

> ```
> {
>   "stats": {
>     "hash": "2a53375ff139d9837e93a38a279d63e5",
>     "bytes": 12345
>   }
> }
> ```

Yet the [Data Resource](https://specs.frictionlessdata.io/schemas/data-resource.json) JSON schema and the [Tabular Data Resource](https://specs.frictionlessdata.io/schemas/tabular-data-resource.json) JSON schema (and specifications) recommend the hash and bytes information to be at the root level.

Question: is this apparent discrepancy intended, or is it something that should be improved?

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by roll** [comment]
@krassowski 
It's a good question. 

It's true that in the Framework we put all the stats in a nested object. It's because of a few reasons including internal performance. The same approach was used in Data Package Pipelines.

Actually, I think that having `stats` as an object is better and more scalable (extensions can add more stats without messing with the root properties) and potentially might be adopted by the specs themselves.

But currently, it contradicts specs so what do you think might be a solution - using a flag on the export?

**Comment by roll** [comment]
FIXED in v5 (#1119) (will be released this month)

```bash
frictionless describe data/table.csv --stats --standards v1
```

or

```python
from frictionless import Resource, system

system.standards_version = 'v1' # or with system.use_standards_version('v1')
resource = Resource('table.csv')
resource.infer(stats=True)
resource.to_descriptor()
```
